modules/calendar/html_parser.py

```diff
@@ -3,9 +3,10 @@
 
 __MD_LINK_REGEX = re.compile(
 	# Group 1: The label
-	# Group 2: The full URL
+	# Group 2: The full URL (optionally title text as well)
 	r"\[(.*?[^\\])\]\((https?:\/\/.*?)\)"
```

Comment:
```suggestion
	# Group 3: The full URL without a title
	r"\[(.*?[^\\])\]\(((https?:\/\/.*?)\s+.*?)\)"
```
In __md_link_to_html you can use group 3 instead of 2, which will ignore the user's title rather than mess up the link